src/Core/tests/Benchmarks.Droid/MainInstrumentation.cs

```diff
@@ -67,20 +53,4 @@ static bool Run()
 		}
 		return success;
 	}
-
-	// NOTE: the built-in ConsoleLogger throws PlatformNotSupportedException
-	class Logger : ILogger
```

Comment:
Was there a fix in BDN for the logger? My memory is that some `System.Console` API threw a `PlatformNotSupported` exception on Android. I don't remember if I sent a PR for it.

If you ran this on Android and it worked, we're good. I think it threw immediately when benchmarks started.